### PR TITLE
Add recent works about sections

### PR DESCRIPTION
**Pull request by Drod86**
<h2>In this branch I:</h2>

- Added markup for recent-works section
- Added markup for about section
- Styled recent-works section
- Styles about section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,63 +20,197 @@
...
+          </article>
+          <article class="card">
+            <h3 class="heading">Profesional Art Printing Data</h3>
+            <p class="blurb">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="badges">
+              <li class="badge">HTML</li>
+              <li class="badge">Bootstrap</li>
+              <li class="badge">Ruby</li>
+            </ul>
+            <button type="button" class="btn">See project</button>
```
Comment (ShadyShawkat) [inline index.html:167]: - [ ] Please make sure that all buttons have their interactions styles set up, to better match the Figma template.

image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Drod86, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Drod86** [comment]
Hello @ShadyShawkat. Thank you for pointing out that I omitted the button event styles. I made the changes.

**Review by skyv26 — APPROVED** [review]
## Hi @Drod86, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅  
- Professional `README` template ✅ 
- Linters are green ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Drod86** [comment]
Hi @ShadyShawkat. It looks like the linter tests have passed.

**Pull request merged**